runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBoxOverlay.java

```diff
@@ -125,14 +125,30 @@ public Dimension render(Graphics2D graphics)
 			.right(StackFormatter.quantityToRSDecimalStack(rightNum, true))
 			.build();
 
-		final LineComponent xpHour = LineComponent.builder()
-				.left("XP/Hour:")
-				.right(StackFormatter.quantityToRSDecimalStack(snapshot.getXpPerHour(), true))
+		final String leftBottomStr;
```

Comment:
Typically directions are spoken and written as bottom-right, so `bottomLeftStr`?

Reply:
Updated